src/v2/spec.yaml

```diff
@@ -2176,6 +2176,35 @@ components:
                 $ref: "#/components/schemas/CastWithInteractions"
         next:
           $ref: "#/components/schemas/NextCursor"
+    FrameNotificationTokens:
+      type: object
+      required:
+        - notification_tokens
+        - next
+      properties:
+        notification_tokens:
+          type: array
+          items:
+            type: object
+            properties:
+              object:
+                type: string
+                enum:
+                  - notification_token
+              token:
+                type: string
+              status:
```

Comment:
status should be enum of values

Reply:
based on code search, i added enabled and disabled statuses